build/templates/session.py.mako

```diff
@@ -126,7 +126,7 @@ init_call_params = helper.get_params_snippet(init_function, helper.ParameterUsag
 
     def __setattr__(self, key, value):
         if self._is_frozen and key not in dir(self):
-            raise TypeError("%r is a frozen class" % self)
+            raise AttributeError("'Session' object has no attribute %r" % key)
```

Comment:
From the Python side of things, the proper term is "property" not attribute. Please use that.
Also, use .format() which is what we've been using everywhere rather than the old printf style syntax.

Reply:
I stand corrected.

>>> class Foo:
...  a = 2
... 
>>> f = Foo()
>>> f.a
2
>>> f.b
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: Foo instance has no attribute 'b'
>>>